Mathlib/Topology/Order/ScottTopology.lean

```diff
@@ -367,8 +376,8 @@ end Scott
 variable [Preorder α]
 
 lemma scottHausdorff_le_lower : scottHausdorff α ≤ lower α :=
-  fun s h => @IsScottHausdorff.isOpen_of_isLowerSet  _ _ (scottHausdorff α) _ _
-      <| (@IsLower.isLowerSet_of_isOpen (Topology.WithLower α) _ _  _ s h)
+  fun s h => @IsScottHausdorff.isOpen_of_isLowerSet _ _ (scottHausdorff α) _ _
```

Comment:
```suggestion
  fun s h => IsScottHausdorff.isOpen_of_isLowerSet (α := scottHausdorff α)
```
or something like that?

Reply:
`(t := scottHausdorff α)` works if I change `[TopologicalSpace α]` to `[t : TopologicalSpace α]` higher up.